core/includes/theme.inc

```diff
@@ -2056,6 +2056,11 @@ function theme_breadcrumb($variables) {
  *       only applies to one column at a time.
  *     - Any HTML attributes, such as "colspan", to apply to the column header
  *       cell.
+ *   - footer: An array of table rows which will be printed within a <tfoot>
```

Comment:
Does this also need an `@since `?

Reply:
It does @indigoxela, but I need to first know if this is going to be in 1.19.0 or 1.18.4.

Perhaps one of our core committers can add that as they merge this?